app/handlers/admin/banners_manage.rb

```diff
@@ -0,0 +1,41 @@
+module Admin
+  class BannersManage
+
+    lev_handler
+
+    paramify :banner do
+      attribute :message, type: String
+      validates :message, presence: true
+    end
+
+    protected
+
+    def authorized?
+      true
+    end
+
+    def handle
+      expires_at_central = string_from_date_select_params(params[:banner], :expires_at)
+      expires_at = central_to_utc(expires_at_central)
+
+      outputs[:banner] = Banner.where(id: params[:id]).first_or_initialize.tap { |banner|
+        banner.message = params[:banner][:message]
+        banner.expires_at = expires_at
+        banner.save
+      }
```

Comment:
I prefer `do...end` for multiline blocks but you don't have to change this because a lot of people don't care anyway :)

Reply:
Oh. Yeah, I also prefer that, I just changed it because I wanted to paste it into the console all as one line and forgot to change it back. Fixed.